stakemachine/__main__.py

```diff
@@ -68,12 +66,27 @@ def main() :
         for name in config["bots"]:
             config["bots"][name]["markets"] = markets
 
+    if (("wif" not in config or not config["wif"]) and "wif" in os.environ):
+        wif = os.environ["wif"]
+        config["wif"] = wif
+
     if (("wallet_host" not in config or not config["wallet_host"]) and
-            ("wif" not in config or not config["wif"])):
+            ("account" not in config or not config["account"])):
         raise Exception(
-            "Need either a wif key or connection details for to the cli wallet."
+            "Need either an account (name) or connection details for to the cli wallet."
```

Comment:
The rpc call to get an account from a pubkey is called `get_key_references(pubkey)` it returns an array of accounts that use that pubkey